askup/templates/askup/qset_modals.html

```diff
@@ -0,0 +1,108 @@
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
+                <h4 class="modal-title">Edit Qset</h4>
+            </div>
+            <div class="modal-body">
+                <form accept-charset="UTF-8" action="/qset/{{qset_id}}/" class="edit_qset" id="edit_qset_{{qset_id}}" method="post">
+                    <div style="display:none">
+                        <input name="utf8" type="hidden" value="✓">
+                        <input name="_method" type="hidden" value="patch">
+                        <input name="authenticity_token" type="hidden" value="M4fIPRA7e5+prdJH5iKHEtoNgQmoIlCdY4/Ix0pl69I=">
+                    </div>
+                    <div class="form-group">
+                        <label for="qset_name">Change the name of this qset</label>
+                        <input id="qset_name" name="qset[name]" type="text" value="{{qset_name}}">
+                    </div>
+                    <div class="form-group">
+                        <strong>Qset permissions</strong>: note that these permissions apply only to non-admin users,
+                        and clicking 'Save' will change the permissions for this Qset and <strong>all of its children</strong>,
+                        without checking whether any permissions values have changed.
+                    </div>
+                    
+                    <div class="form-group">
+                        <label for="qset_permissions_all_questions_visible">Questions are visible to all logged-in users</label>
+                        <input name="qset[permissions][all_questions_visible]" type="hidden" value="false">
+                        <input checked="checked" id="qset_permissions_all_questions_visible" name="qset[permissions][all_questions_visible]" type="checkbox" value="true">
+                    </div>
+                    <div class="form-group">
+                        <label for="qset_permissions_question_authors_visible">Questions authors are visible to all users</label>
+                        <input name="qset[permissions][question_authors_visible]" type="hidden" value="false">
+                        <input id="qset_permissions_question_authors_visible" name="qset[permissions][question_authors_visible]" type="checkbox" value="true">
+                    </div>
+                    <div class="form-group">
+                        <label for="qset_permissions_questions_visible_to_unauth_user">Questions are visible to unauthenticated users</label>
+                        <input name="qset[permissions][questions_visible_to_unauth_user]" type="hidden" value="false">
+                        <input checked="checked" id="qset_permissions_questions_visible_to_unauth_user" name="qset[permissions][questions_visible_to_unauth_user]" type="checkbox" value="true">
+                    </div>
+                    <div class="form-group">
```

Comment:
The template should be rewritten with the Django template forms.

Reply:
It will be rewritten in the next commits, when we're going to Implement an actual Django model forms for this UI parts. It was transferred from the previous project and didn't commented for the sake of keeping the Bootstrap functional.